pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -107,6 +112,7 @@ func NewDesiredStateOfWorldPopulator(
 		hasAddedPodsLock:         sync.RWMutex{},
 		csiMigratedPluginManager: csiMigratedPluginManager,
 		intreeToCSITranslator:    intreeToCSITranslator,
+		pvcBeingDeleted:          make(map[*v1.PersistentVolumeClaim]bool),
```

Comment:
I find it a little confusing that we're using the name `pvcBeingDeleted` twice - one for the constant string and the other time for a field in the struct. Could we please change one of them?